src/bin/lfortran.cpp

```diff
@@ -2232,6 +2212,7 @@ int main_app(int argc, char *argv[]) {
     app.add_option("--skip-pass", skip_pass, "Skip an ASR pass in default pipeline");
     app.add_flag("--show-llvm", show_llvm, "Show LLVM IR for the given file and exit");
     app.add_flag("--show-mlir", show_mlir, "Show MLIR for the given file and exit");
+    app.add_flag("--show-mlir-to-llvm", show_mlir_to_llvm, "Show LLVM IR after translating from MLIR for the given file and exit");
```

Comment:
```suggestion
    app.add_flag("--show-mlir-to-llvm", show_mlir_to_llvm, "Show LLVM IR translated from MLIR for the given file and exit");
```

Does this look good to you?

Reply:
Yes.